src/index.js

```diff
@@ -1,39 +1,40 @@
 const express = require("express");
 const logger = require("morgan");
-const  cors = require("cors");
+const cors = require("cors");
 const { startApp } = require("./databaseSetup");
 const ourLogger = require("./logger");
 
 const testRouter = require("./components/test");
-const usersRouter = require("./routes/users")
+const usersRouter = require("./routes/users");
 const categoryRouter = require("./routes/categories");
-const eventsRouter = require('./routes/events');
-const unitofmeasurings = require('./routes/unitofmeasurings.js');
+const eventsRouter = require("./routes/events");
+const accountRouter = require("./routes/account");
+const unitofmeasurings = require("./routes/unitofmeasurings.js");
 const adminContactsRoutes = require("./routes/contactAdmin");
 
 const app = express();
 
-const formatsLogger = app.get('env') === 'development' ? 'dev' : 'short';
+const formatsLogger = app.get("env") === "development" ? "dev" : "short";
```

Comment:
Replace all double quotes with single ones

Reply:
oh, okay

Reply:
why it need? We always use double.

Reply:
fixed